### PR TITLE
Kubelet code cleanup

### PR DESCRIPTION
**Pull request by bart0sh**

### CODE REVIEW

File: pkg/kubelet/cm/dra/manager.go [hunk 1]
```diff
@@ -88,36 +97,49 @@ func (m *ManagerImpl) prepareContainerResources(pod *v1.Pod, container *v1.Conta
...
 
 			// Query claim object from the API server
-			resourceClaim, err := m.kubeClient.CoreV1().ResourceClaims(pod.Namespace).Get(context.TODO(), claimName, metav1.GetOptions{})
+			resourceClaim, err := m.kubeClient.CoreV1().ResourceClaims(pod.Namespace).Get(
+				context.TODO(),
+				claimName,
+				metav1.GetOptions{})
 			if err != nil {
-				return fmt.Errorf("failed to fetch ResourceClaim %s referenced by pod %s: %+v", claimName, pod.Name, err)
+				return fmt.Errorf("failed to fetch ResourceClaim %s referenced by pod %s: %w", claimName, pod.Name, err)
```
Comment (pohly) [inline pkg/kubelet/cm/dra/manager.go]: What's the rationale for wrapping the error? Why is that better and/or needed?

See "Whether to Wrap" in https://go.dev/blog/go1.13-errors

Comment (bart0sh) [reply pkg/kubelet/cm/dra/manager.go]: The rationale was to fix this golangci warning:
```
pkg/kubelet/cm/dra/manager.go:110:106: non-wrapping format verb for fmt.Errorf. Use `%w` to format errors (errorlint)
                                return fmt.Errorf("failed to fetch ResourceClaim %s referenced by pod %s: %+v", claimName, pod.Name, err)
```

Comment (pohly) [reply pkg/kubelet/cm/dra/manager.go]: That check is too strict. As explained in the design document for errors, there are cases where errors should not get wrapped. This is part of the API design of a package and something that a developer needs to decide on a case-by-case basis.

Comment (bart0sh) [reply pkg/kubelet/cm/dra/manager.go]: should be fixed now. please, reivew

**Comment by pohly** [comment]
I've manually merged the last commit and squashed it.

**Pull request closed without merging**